src/coreclr/src/pal/src/exception/seh.cpp

```diff
@@ -322,7 +322,7 @@ PAL_ERROR SEHEnable(CPalThread *pthrCurrent)
 {
 #if HAVE_MACH_EXCEPTIONS
     return pthrCurrent->EnableMachExceptions();
-#elif defined(__linux__) || defined(__FreeBSD__) || defined(__NetBSD__)
+#elif defined(__linux__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__sun)
     // TODO: This needs to be implemented. Cannot put an ASSERT here
```

Comment:
Since you are touching this code, could you please remove this obsolete comment?